extension/encodingextension/README.md

```diff
@@ -0,0 +1,63 @@
+# Encoding extension
+
```

Comment:
Do you want to generate the status box?

Reply:
Yeah, sorry message on slack to add the details.

Reply:
Yeah, sorry message on slack to add the details.